WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -1191,19 +1194,38 @@ private void showQuickStartDialog() {
             return;
         }
 
+        @StringRes int titleRes;
+        @StringRes int messageRes;
+        @StringRes int positiveButtonTitleRes;
+        @StringRes int negativeButtonTitleRes;
+        @StringRes int neutralButtonTitleRes = -1;
+
+        if (mOnboardingImprovementsFeatureConfig.isEnabled()) {
+            titleRes = R.string.quick_start_dialog_need_help_manage_site_title;
```

Comment:
TBH, I thought about it, my reasoning: 

I named new promo dialog to have `_new` suffix with the same intention. For the strings, I wanted to finalize their keys as changing them later will cause another round of translations. So that we can identify new strings, I added a block comment with the "old" and "new" mention. 

Let me know if you strongly feel about having `_new` suffix temporarily for strings.

Reply:
👋 @ashiagr !

Your reasoning makes sense, and yea, I agree, I think finalizing the string now and not having them being translated again seems very reasonable. Let's keep it as is. 🙏